components/application-gateway/internal/authorization/csrf/csrf.go

```diff
@@ -0,0 +1,73 @@
+package csrf
```

Comment:
I'm just wondering if the directory structure is correct. "CSRF" directory is under "authorization". Looking at the proxy code we treat CSRF as something else, not an authorization method. Maybe there should be "internal/csrf" folder?

Reply:
As authorization is required to fetch CSRF tokens, we decided to place our code under authorization dir.

Reply:
Well, there is dependency between CSRF tokens and authorisation but code in proxy package considers it as something else. CSRF is a method of protecting from certain type of attack rather than separate method of authorization.

Reply:
@akgalwas Good point, package is moved.